### DISCUSSION TITLE
[解答]欢迎来问四糸乃酱问题，绳网靠大家~

### DISCUSSION THREAD

**Discussion by lovelolit** [share121/inter-knot#1137] [其他]
### 欢迎询问各类问题，本绳匠会根据自身工作经验以及空洞经历，解答留言问题~
**个人观点，仅供参考**

包括但不限于：
1. 电影、视频、小说、游戏。
2. 生活、学习问题。
3. 大学生活、学习与考保研内容。
4. 委托任务。

**注意事项**

1. 禁止涉及敏感问题以及故意为难。
2. 不帮写作业！
3. 本绳匠涉猎很广，不懂也会陪你一起解决！
4. 最终解释权归四糸乃酱所有。

![o_1di9o3vch1g8d1gefdjo1gq1vqir](https://github.com/user-attachments/assets/71809209-6f36-413a-ac52-d098c37acb88)

**Comment by chuanqishengjiang** [comment]
よしの酱卡哇伊~

**Reply by lovelolit** [reply]
ty ;D 
![QQ图片20240812121128](https://github.com/user-attachments/assets/2db1fcfa-d065-4113-9354-6efc81d48659)

**Comment by chuanqishengjiang** [comment]
可以把四糸奈酱给我摸摸吗！

**Reply by lovelolit** [reply]
不阔以

**Comment by shenguojia123** [comment]
根号16的平方根是多少

**Reply by lovelolit** [reply]
± 2! (‾◡◝)

**Comment by capacitor1** [comment]
啊 真心想问一个游戏问题：

**我在玩MC的时候，点了64x64x64个TNT，此时游戏帧数降到了1FPS，但是我CPU占用率只有10%？？只有3个核心满载，其余全是空的**

这是游戏的优化不好还是游戏/java/cpu不适配？或者就是正常现象？怎么解决？

> 注：之前用win10和java17.被网友教导win10多核优化不好，于是更换了java21和win11. java22我用不了会报错

附：
系统：win11 23h2
java：21.0.3
MC：1.20.1 forge47.0.34 optifineI5，打开了光影
cpu：i9-14900k

**Reply by lovelolit** [reply]
过多TNT掉帧正常，最大影响应该是光影，其次是游戏设置。个人认为是正常现象。
你可以用专业测评的软件看具体数据，CPU占有率10不正常。可以上网调整设置。我不精通电脑，期待评论区大佬咯。
JAVA我一直都是根据版本来的，直接用PCL2启动器，没有报错过。

**Comment by soyobat** [comment]
请问用jwt和session做登录，那个技术选型优势比较大？

**Reply by lovelolit** [reply]
JWT:无需在服务器端保存会话信息，减少了服务器的内存开销，易于扩展，支持跨域认证。
缺点：由于JWT是无状态的，所以在使用JWT注销时，服务器不能直接作出反应，因为它并不保存任何会话信息。
session:可以轻易注销会话，可以在服务器端保存用户的状态信息。
缺点：需要在服务器端保存会话信息，增加了服务器的内存开销，不适合分布式或者无状态的系统。
问到知识盲区了(｡◝ᴗ◜｡)。GPT，启动！

**Comment by dengyunzhe** [comment]
四糸乃酱のパンツの色は何ですか？

**Reply by lovelolit** [reply]
だめ，hentaio(≧口≦)o
